kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -114,7 +114,9 @@ grCollectionsPanel.controller('GrNodeCtrl',
     };
 
     subscribe$($scope, pathWithImages$, ({path, images}) => {
-       collections.addImagesToCollection(images, path).then(() => ctrl.saving = false);
+       collections.addImagesToCollection(images, path).then(() => {
+           $scope.$apply(() => ctrl.saving = false);
```

Comment:
Good point, that may be it as we don't `$apply` in the [`subscribe$` helper](https://github.com/guardian/grid/blob/6bd20d7a73a8626acd3a9d9ab7e18568d866aa3a/kahuna/public/js/util/rx.js#L30-L35), even though I would still have expected it to apply through the use of `$http` and `$q`. Might be worth trying adding the `$apply` there to fix the more general problem (though such a change would need thorough QA as it has wide implications).